build/providers/facebook/android/gradletealeaf.xml

```diff
@@ -1,3 +1,3 @@
 //<!-- Facebook -->
-  implementation 'com.ironsource.adapters:facebookadapter:4.3.2@jar'
-  implementation 'com.android.support:recyclerview-v7:25.3.1'
```

Comment:
The build.gradle is also changing according to this https://developers.ironsrc.com/ironsource-mobile/android/mediation-networks-android/#step-1

Can you verify for others too

Reply:
Also please verify if manifest has changes too